### ISSUE TITLE
GSoC 2016 Integrate with fabric.io to track crashes

### ISSUE THREAD

**Issue by bphenriques** [systers/malaria-app-ios#28]
https://get.fabric.io/

Track crashes: Whenever the app crash, an issue is published on this github project with the description of the crash.

**Comment by bphenriques** [comment]
In my opinion these are the essential requirements:
- Track user id (Random UUID)
- Track custom events (in order to trace the errors)
- Setting to disable analytics (for privacy)

**Comment by manas15** [comment]
Umm so basically what we need is a crash report generator. Each time we encounter an app crash the app will send a detail report to the our dashboard of fabric.io ( which will have a unique identifier UUID ). Right ?

**Comment by bphenriques** [comment]
I am not aware of the best practices here (due to privacy policies) so I would like to hear more opinions and talk to a more experienced iOS developers:
- Assign a unique id to the iOS app installation
- As you say, unique id to the crash. If so, I don't know how that could be useful

Maybe I am making this a big issue and the unique identifiers aren't really needed. However, I know that Peace Tracks would like to track its volunteers pill intake so... in that case there will be an id associated with the app.

**Comment by n0shake** [comment]
Hello guys,

My name is Abhishek. I am a student studying Computer Science at the University of Florida.

I have implemented Crashlytics in my previous OS X app. I have attached the screenshot of my dashboard; there is know need to assign crash IDs per say, but we can pass user-specific information. Crash caused due to the same bug are grouped by number of users.

<img width="1151" alt="screen shot 2016-03-17 at 10 49 04 am" src="https://cloud.githubusercontent.com/assets/8280282/13849709/aa095b36-ec2d-11e5-8b36-0c5d4f818463.png">
